manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```diff
@@ -230,6 +238,11 @@ spec:
         - name: shared-doca-driver-toolkit
           emptyDir: {}
         {{- end }}
+        {{- if .CrSpec.DriverInventoryPath }}
```

Comment:
do we have an explicit requirement to make this configurable ?

Reply:
No, we can also add this a true/false switch

Reply:
do you (or others) see issue with always defining the volume and its volume mount ?

then we can have the env variable set by default (there is some code in state that handles default env vars)
if we need to disable for some reason, user can override this env var with empty string

Reply:
Sounds good to me. Should be documented, otherwise I see no issues with this approach.

@rollandf @ykulazhenkov @e0ne @killianmuldoon @vasrem what do you think?

Reply:
I agree that we should hide things from user where possible.
Also, need to verify on OpenShift where this dir should be created and if it will be kept after reboot. (Maybe there is no issue)

Reply:
> then we can have the env variable set by default (there is some code in state that handles default env vars)
if we need to disable for some reason, user can override this env var with empty string

Set by default in the code not in helm values, right? +1 to handle this in the code.

Reply:
> Set by default in the code not in helm values, right? +1 to handle this in the code.
yes

> Also, need to verify on OpenShift where this dir should be created
we can ask RH folks in the weekly meeting. in my openshift single node cluster with coreOS, /opt is mapped to /var/opt 
this is the mount im seeing (not tmpfs):
/dev/sda4 /var xfs rw,seclabel,relatime,attr2,inode64,logbufs=8,logbsize=32k,prjquota 0 0

Reply:
So far I've deduced that /var should be writable. We could create this dir in /var/opt if the RH folks confirm it's indeed writable with privileged SCC

Reply:
Updated the PR and used `/var/opt/ofed-inventory` dir